docs/data/styles.json

```diff
@@ -48,17 +48,17 @@
                 "items": [
                     {
                       "title": "Sizes",
-                      "url": "",
+                      "url": "#spacing",
```

Comment:
I'd rather link to the top of the Spacing section to show the header since the nav headers are not clickable.